ext/scripts/security_scan/run_oyster.sh

```diff
@@ -12,4 +12,9 @@ SCRIPT_DIR="$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")"
 output_path=$1
 
 echo Start oysteR...
-Rscript "$SCRIPT_DIR/exec_oyster.R" "$output_path" &> "$output_path/oyster_debug.log" && echo "No vulnerabilities found!" || (cat "$output_path/oyster_debug.log" && exit 1)
+if [[ $(Rscript "$SCRIPT_DIR/exec_oyster.R" "$output_path" &> "$output_path/oyster_debug.log") ]]; then
+  echo "No vulnerabilities found!"
+else
+  cat "$output_path/oyster_debug.log"
+  exit 1
+fi
```

Comment:
Reason:
 ^-- SC2015: Note that A && B || C is not if-then-else. C may run when A is true.

Reply:
One reason why I prefer if else. The shortcut evaluation is tricky.